[이성보]구구단 예제입니다

ExiledI opened the pull request.

ExiledI committed:
[이성보]구구단 예제입니다

yshush commented on src/firstclass.java:
```diff
@@ -0,0 +1,32 @@
+			System.out.println("�߸�� ���ڸ� �Է� �ϼ̽�ϴ� �ٽ��Է����ּ���");
+			c = sc.nextInt();
+		}	// 1~999 �̿ܿ��� ��� �ݷ���
+		//��� �κ�//
```
여기는 for문을 써서 출력해보시는게 어떨까요?
더불어 클래스 명은 영대문자로 시작하는 파스칼케이스로 작성하시는게 좋습니다!

ExiledI committed:
[이성보]구구단 과제 제출입니다

전에는 for문을 안써서 for문을 사용 해보았고
다른 사람들의 과제 내용도 참고해서 예외 던지기 클래스 생성도 해봤는데
private static final long serialVersionUID = 1L; 이문구는 뭘뜻하는건가요 언뜻봐서는 메인 클래스랑 연결해주는 의미인거같은데 정확히뭔지 모르겠습니다. 이게 없으면 제대로 안돌아가더라고요..

ExiledI committed:
[이성보]구구단 계산기 입니다

원래는 mvc모델로 나누어서 코딩하려고 했는데 클래스 나누어서 만드는게 잘 안되서 처음 만든 합쳐진 버전 올립니다.

ExiledI committed:
[이성보]2시 50분까지 진행상황입니다

mvc 로 클래스파일을 무작정 만들고 채워넣는식으로 만드는중입니다
